packages/typeorm/src/utils.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright IBM Corp. 2017,2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {JsonSchemaOptions, SchemaRef} from '@loopback/rest';
+
+/**
+ * Describe the provided Entity as a reference to a definition shared by multiple
+ * endpoints. The definition is included in the returned schema.
+ *
+ * @param entityCtor - The entity constructor (e.g. `Product`)
+ * @param options - Additional options
+ */
+export function getModelSchemaRef<T extends object>(
```

Comment:
As discussed in the call, I am proposing to drop `Ref` part and emit regular JSON schema. The `ref`-based schemas were needed before OpenAPI schema consolidation was introduced. I believe with consolidation in place, we no longer need to worry about avoiding duplicated schema definitions in controller methods.

```ts
export function modelToJsonSchema<T extends object>(
  ctor: Function & {prototype: T},
  jsonSchemaOptions: JsonSchemaOptions<T> = {},
): JsonSchema {
  const allColumns: ColumnMetadataArgs[]= getMetadataArgsStorage().columns;
  const modelColumns = allColumns.filter(c => c.target === modelCtor);
  const schema: JsonSchema = {
    // set title (to modelCtor.modelName?), etc.
    properties: {}
  };
  for (c of modelColumns) {
    schema.properties[c.propertyName] = {
      type: c.options.type,
    }
  }
  return schema;
}
// getJsonSchema(modelCtor) --> call modelToJsonSchema under the hood + cache the result
// getModelSchema(modelCtor) --> convert getJsonSchema() to OpenApiSpec format
```

Reply:
> As discussed in the call, I am proposing to drop `Ref` part and emit regular JSON schema. The `ref`-based schemas were needed before OpenAPI schema consolidation was introduced. I believe with consolidation in place, we no longer need to worry about avoiding duplicated schema definitions in controller methods.

❗I'd like you to address my comment before the PR is landed.

Reply:
If it's too much work to get even a minimal conversion working, then I am proposing the following:

- rename `getModelSchemaRef` to `getModelSchema` 
- change the return type from `SchemaRef` to `SchemaObject`
- change the implementation to return a "dummy" schema describing any object value

```ts
export function getModelSchemaRef<T extends object>(
  entityCtor: Function & {prototype: T},
  options?: JsonSchemaOptions<T>,
): SchemaRef {
  // TODO: generate proper OpenAPI schema describing entity properties, etc.
  return {
    title: entityCtor.name,
    type: 'object'
  };
}
```

Personally, I'll try to get rid of `SchemaRef` (and thus the dependency on `@loopback/rest`) by removing the description of the return type or using `object` instead. For the first iteration, we don't need `options` either, since they are not supported anyways.

```ts
export function getModelSchemaRef<T extends object>(
  entityCtor: Function & {prototype: T},
) {
  return { 
    // ...
  };
}
```